modules/apps/portal-search/portal-search-ranking-web/src/main/resources/META-INF/resources/js/components/list/Item.es.js

```diff
@@ -563,13 +563,15 @@ class Item extends PureComponent {
 						)}
 					</div>
 
-					<ItemDropdown
-						addedResult={addedResult}
-						hidden={hidden}
-						onClickHide={this._handleHide}
-						onClickPin={this._handlePin}
-						pinned={pinned}
-					/>
+					{onClickPin && onClickHide &&
+						<ItemDropdown
+							addedResult={addedResult}
+							hidden={hidden}
+							onClickHide={this._handleHide}
+							onClickPin={this._handlePin}
+							pinned={pinned}
+						/>
+					}
```

Comment:
Can we do this check inside `ItemDropdown` itself? If `onClickHide` isn't available, we shouldn't hide this whole dropdown since `onClickPin` should still be possible.